src/mmw/apps/home/views.py

```diff
@@ -116,6 +116,10 @@ def set_url(layer):
 
 
 def get_client_settings(request):
+    # BiG-CZ mode applies when either request host contains predefined host, or
+    # ?bigcz=true query parameter is present. This covers staging sites, etc.
+    bigcz = (settings.BIGCZ_HOST in request.META.get('HTTP_HOST') or
+             request.GET.get('bigcz') == 'true')
```

Comment:
At first I had it set to unset the session variable on `?bigcz=false`, but since we're not doing that anymore I'll update this to just check for its presence.